### PR TITLE
Show screenshot descriptions in image carousel

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

**Comment by szakarias** [comment]
image

File: app/lib/frontend/templates/views/pkg/screenshots.dart [hunk 1]
```diff
@@ -60,11 +63,15 @@ d.Node collectionsIcon() {
 }
 
 d.Node screenshotThumbnailNode(
-    String thumbnailUrl, List<String>? screenshotUrls) {
+  String thumbnailUrl,
+  List<String>? screenshotUrls,
+  List<String>? screenshotDescriptions,
```
Comment (isoos) [inline app/lib/frontend/templates/views/pkg/screenshots.dart:68]: Nit: I think at this point all of these could be named parameters.

File: app/lib/frontend/templates/views/pkg/screenshots.dart [hunk 2]
```diff
@@ -60,11 +63,15 @@ d.Node collectionsIcon() {
...
+) {
   return d.div(classes: [
     'thumbnail-container'
   ], attributes: {
     'data-thumbnail': screenshotUrls!.join(','),
+    'data-thumbnail-descriptions': screenshotDescriptions!.join(','),
```
Comment (isoos) [inline app/lib/frontend/templates/views/pkg/screenshots.dart:74]: what happens if a description has a comma in it?

Comment (isoos) [reply app/lib/frontend/templates/views/pkg/screenshots.dart:74]: Maybe this should be a `data-thumbnail-description-N` attribute, or something json-encoded?

File: app/lib/frontend/templates/views/pkg/screenshots.dart (hunk 1 above)
Comment (szakarias) [reply app/lib/frontend/templates/views/pkg/screenshots.dart]: done

File: app/lib/frontend/templates/views/pkg/screenshots.dart (hunk 2 above)
Comment (szakarias) [reply app/lib/frontend/templates/views/pkg/screenshots.dart]: done!

**Review by isoos — APPROVED** [review]

**Pull request merged**